docs/run-platform-server/ingestion-filtering.mdx

```diff
@@ -5,7 +5,7 @@ order: 46
 
 ## Overview
 
-Ingestion Filtering enables Horizon operators to drastically reduce the storage footprint of the historical data in the Horizon database by whitelisting Assets and/or Accounts that are relevant to their operations.
+Ingestion Filtering enables Horizon operators to drastically reduce the storage footprint of the historical data in the Horizon database by allow-listing Assets and/or Accounts that are relevant to their operations.
```

Comment:
Although I initially suggested changing 'whitelist' to 'allowlist,' I now realize that the API response already uses the term like this `{"whitelist":["accountid1","accountid2","accountid3"],"enabled":true,"last_modified":1689048762}` So, should we stick to the same terminology in the document as well? I apologize for any extra work this may have caused. Thanks!

Reply:
ah, nice catch, np, will change that back for consistency, it was just a couple places specific to horizon docs